### PR TITLE
trackcards

### PR DESCRIPTION
**Pull request by fgottesmann**
My first pull request with my card component.

### CODE REVIEW

File: components/cards/Cards.module.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
+.trackcard {
+  border-radius: 100px;
+  border: 1px solid #2e2d2c;
+  background-color: #2e2d2c;
+  -webkit-box-shadow: -3px -3px 6px #d6d6d6;
+  -moz-box-shadow: -3px -3px 6px #d6d6d6;
```
Comment (phlgr) [inline components/cards/Cards.module.css]: Those will get generated automatically

File: components/cards/Cards.module.css [hunk 2]
```diff
@@ -0,0 +1,53 @@
+.trackcard {
+  border-radius: 100px;
```
Comment (phlgr) [inline components/cards/Cards.module.css:2]: If you want the track card to be perfectly round you could use `50%`. A deployment would be helpful here

File: components/cards/Cards.module.css [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  border: 1px solid #2e2d2c;
+  background-color: #2e2d2c;
+  -webkit-box-shadow: -3px -3px 6px #d6d6d6;
+  -moz-box-shadow: -3px -3px 6px #d6d6d6;
+  box-shadow: -3px -3px 6px #d6d6d6;
+  max-width: 95%;
+  max-height: 12rem;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
+    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
```
Comment (phlgr) [inline components/cards/Cards.module.css]: Why do you set these here?

File: components/cards/Cards.module.css [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  grid-template-rows: 1fr 1fr;
+  grid-template-areas:
+    "cardimage cardimage title title title likebutton"
+    "cardimage cardimage artist artist artist likebutton";
+}
+.cardimage {
+  grid-area: cardimage;
+  width: 100%;
+  height: 100%;
+  border: 2px solid #ec9e17;
```
Comment (phlgr) [inline components/cards/Cards.module.css]: You could use color variables here

File: components/cards/Cards.module.css [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+  border-radius: 100px;
+  object-fit: cover;
+  box-shadow: -3px -3px 6px #ec9e17;
+}
+.title {
+  grid-area: title;
+  color: #d6d6d6;
```
Comment (phlgr) [inline components/cards/Cards.module.css]: Color variables

**Review by phlgr — CHANGES_REQUESTED** [review]
👨‍💻 GJ a deployment would be nice

File: components/cards/Cards.module.css (hunk 2 above)
Comment (fgottesmann) [reply components/cards/Cards.module.css:2]: I know that a border radius of 50% makes the image completely round, but i wanted the image to be rounded at the left and right side and also to be stretched which can be seen in fullscreen mode

**Review by phlgr — APPROVED** [review]
GJ 🚀

**Pull request merged**